### DISCUSSION TITLE
Sort the fields of an entity based on desired order

### DISCUSSION THREAD

**Discussion by Archer-Thane** [amplication/amplication#2539] [Ideas]
There will be nice if there was a way to order the fields of an entity, especially on the ui.
Now we have the order based on alphabet. We have the Prisma Schema sorted alphabetically, and ui based on too.
For example if we have a customer entity with fields: ID, username, createdAt, Cart 
then in ui we see this:
Cart, createdAt, ID, username.
That would be nice if there was a way to manage this and change the order of fields to be like this:
ID, username, Cart, createdAt.
That was the idea.
Thanks for the great work.

**Comment by yuval-hazaz** [comment]
Thanks, sound like a good idea

**Comment by yuval-hazaz** [comment]
I will create a feature request from this dicussion
